rootfs-builder/ubuntu/config.sh

```diff
@@ -0,0 +1,29 @@
+# This is a configuration file add extra variables to
+#
+# Copyright (c) 2018  Yash Jain
+#
+# SPDX-License-Identifier: Apache-2.0
+# be used by build_rootfs() from rootfs_lib.sh the variables will be
+# loaded just before call the function. For more information see the
+# rootfs-builder/README.md file.
+#!/bin/bash
+OS_VERSION=${OS_VERSION:-16.04}
+PACKAGES="systemd iptables "
+
+# url to download rootfs from
+ARCHIVE_URL=${ARCHIVE_URL:-"http://archive.ubuntu.com/ubuntu/"}
+#ARCHIVE_URL=${ARCHIVE_URL:-"http://archive.debian.org/debian"}
+
+# this should be ubuntu's codename eg Xenial for 16.04
+OS_NAME=${OS_NAME:-"xenial"}
+
+# packages to be installed by default
+PACKAGES="systemd iptables "
+
+DEBOOTSTRAP=${PACKAGE_MANAGER:-"debootstrap"}
+
+case $(arch) in
+    (x86_64) ARCHITECTURE="amd64";;
+    (ppc64le) ARCHITECTURE= "ppc64el";;
```

Comment:
- Why no arm64 support?
- What is the opening bracket ( `(` ) for in the cases?

Reply:
Oops forgot :disappointed: